esmvalcore/config-developer.yml

```diff
@@ -201,7 +201,7 @@ ACCESS:
     default:
       - '{dataset}/{sub_dataset}/{exp}/{modeling_realm}/netCDF'
```

Comment:
I don't think this will work for ocean data. 
See example in `/g/data/p73/archive/non-CMIP/ACCESS-ESM1-5/ESM-CDR-01/history/ocn`
There is no `Netcdf` folder there.